### PR TITLE
add cvmfs_server resign -p option

### PR DESCRIPTION
**Pull request by DrDaveD**
See [CVM-1140](https://sft.its.cern.ch/jira/browse/CVM-1140)

### CODE REVIEW

File: cvmfs/server/cvmfs_server_resign.sh [hunk 1]
```diff
@@ -35,12 +39,13 @@ cvmfs_server_resign() {
   [ -n "$whitelist_path" ] || check_multiple_repository_existence "$names"
 
   # sanity checks
-  is_root || die "Only root can resign repositories"
+  [ $sign_published -eq 0 ] || [ $require_repo_config -eq 1 ] || die "Cannot use both -n and -p"
```
Comment (jblomer) [inline cvmfs/server/cvmfs_server_resign.sh]: I think this condition refers to an older version of the patch set.

Comment (DrDaveD) [inline cvmfs/server/cvmfs_server_resign.sh]: Good catch, fixed.

File: cvmfs/server/cvmfs_server_resign.sh [hunk 2]
```diff
@@ -35,12 +39,13 @@ cvmfs_server_resign() {
   [ -n "$whitelist_path" ] || check_multiple_repository_existence "$names"
 
   # sanity checks
-  is_root || die "Only root can resign repositories"
+  [ $sign_published -eq 0 ] || [ $whitelist_path -eq 1 ] || die "Cannot use both -w and -p"
```
Comment (jblomer) [inline cvmfs/server/cvmfs_server_resign.sh]: Isn't this still wrong?  The `whitelist_path` parameter carries a string not an integer.

Comment (DrDaveD) [inline cvmfs/server/cvmfs_server_resign.sh]: Yes, sorry, and thank you!  I'm sorry I didn't adequately test it.  Fixed now.

**Pull request merged**